app/controllers/api_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ApiController < ActionController::API
+  rescue_from CanCan::AccessDenied do |exception|
+    redirect_to root_url, alert: exception.message
+  end
+
+  def authorize_request
+    header = request.headers['Authorization']
+    header = header.split.last if header
+    begin
+      @decoded = JsonWebToken.decode(header)
+      @current_user = User.find(@decoded[:user_id])
+    rescue ActiveRecord::RecordNotFound => e
+      render json: { errors: e.message }, status: :unauthorized
+      # rubocop:disable Lint/DuplicateBranch
```

Comment:
- [ ] Kindly remove to keep your code clean, kindly remove ruboco:disable! Also, if you cannot fix the issue that is causing you to use Rubocop you can leave it like that.

Reply:
Thanks for the suggestion! Unfortunately, I can't enable rubocop on these couple of lines since the code is necessary for the app to work with JWT but I can't modify them to keep both proper functionality and pass the current rubocop rules.